inst/rmarkdown/templates/severity/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,326 @@
+---
+title: "Estimating severity and underascertainment"
+author: Carmen Tamayo, Hugo Gruson
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_document:
+    code_folding: "hide"
+link-citations: no
+params:
+  data_file: 
+    label: "Name of file containing cases and deaths over time"
+    value: "data/Marburg_EqGuinea_linelist.csv"
+    input: file
+  data_type:
+    label: "Indicate the type of data to be used in this report"
+    value: "Linelist"
+    choices: ["Linelist", "Aggregated"]
+  disease_name: 
+    label: "Name of the disease of interest, also used to select diseases in the epiparameter database if `use_parameter_database = TRUE`."
+    value: "Ebola"
+  data_columns:
+    label: "Columns present in your dataset"
+    value: "Dates of onset and total deaths only"
+    choices: ["Dates of onset and total deaths only", "Dates of onset and death", "Dates of onset and whether case died"]
```

Comment:
I would probably replace this parameter by two params:

- cases
- deaths

Which can take a column name or a number, and from this, we determine automatically in which situation we are. This means we can skip most of `id_key_data_linelist.Rmd`.

Does this make sense?